service_container/tags.rst

```diff
@@ -359,10 +352,31 @@ To answer this, change the service declaration:
                 tags:
                     - { name: app.mail_transport }
 
-    versionadded:: 3.3
+    .. versionadded:: 3.3
         Support for the compact tag notation in the YAML format was introduced
         in Symfony 3.3.
 
+.. tip::
+
+    In YAML format, you may define a service with a simple array of tags as long
+    as you don't need additional attributes. The following definitions are
+    equivalent.
+
+    .. code-block:: yaml
+
+        services:
+
+            # Compact syntax
+            AppBundle\Twig\AppExtension: [twig.extension]
+
+            # Verbose syntax
+            AppBundle\Twig\AppExtension:
+                tags: [twig.extension]
+
+    .. versionadded:: 3.3
+        Support for the short syntax for service definition in the YAML format
+        was introduced in Symfony 3.3.
```

Comment:
Do we talk about different things? Do you mean the `versionadded` directive? Otherwise I don't see the problem as the changes were only merged into the `master` branch.